### ISSUE TITLE
java.util.MissingResourceException: Registry key node_modules.use.workspace.model is not defined

### ISSUE THREAD

**Issue by dudevo** [jonathanlermitage/intellij-extra-icons-plugin#137]
## Running environment
- Extra Icons plugin version - 2023.2.5-231
- IDE - PhpStorm 2023.1.2
- OS - macOS 13.3.1

## Bug description
Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behaviour.  
Please attach **IDE logs**. Open your IDE and go to <kbd>Help</kbd>, <kbd>Collect Logs and Diagnostic Data</kbd>.

## IDE - additional info
N/A

## IDE - stack trace
```
java.util.MissingResourceException: Registry key node_modules.use.workspace.model is not defined
	at com.intellij.openapi.util.registry.Registry.getBundleValue(Registry.java:169)
	at com.intellij.openapi.util.registry.RegistryValue._get(RegistryValue.java:234)
	at com.intellij.openapi.util.registry.RegistryValue.get(RegistryValue.java:204)
	at com.intellij.openapi.util.registry.RegistryValue.asBoolean(RegistryValue.java:63)
	at com.intellij.openapi.util.registry.Registry.is(Registry.java:63)
	at com.intellij.javascript.nodejs.library.NodeModulesDirectoryManager.shouldUseWorkspaceModel(NodeModulesDirectoryManager.java:874)
	at com.intellij.javascript.nodejs.library.NodeModulesExcludePolicy.getExcludeUrlsForProject(NodeModulesExcludePolicy.java:28)
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.computeCustomExcludedRoots(NonIncrementalContributors.kt:99)
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.updateIfNeeded(NonIncrementalContributors.kt:48)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.ensureIsUpToDate(WorkspaceFileIndexDataImpl.kt:128)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.getFileInfo(WorkspaceFileIndexDataImpl.kt:70)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.getFileInfo(WorkspaceFileIndexImpl.kt:220)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.isUnderIgnored(ProjectFileIndexImpl.java:99)
	at com.intellij.openapi.roots.impl.ProjectFileIndexFacade.isUnderIgnored(ProjectFileIndexFacade.java:66)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.isExcludedOrIgnored(FileManagerImpl.java:425)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findDirectoryImpl(FileManagerImpl.java:411)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findDirectory(FileManagerImpl.java:403)
	at com.intellij.psi.impl.PsiManagerImpl.findDirectory(PsiManagerImpl.java:169)
	at com.intellij.psi.AbstractFileViewProvider.shouldCreatePsi(AbstractFileViewProvider.java:86)
	at com.intellij.psi.SingleRootFileViewProvider.createFile(SingleRootFileViewProvider.java:149)
	at com.intellij.psi.SingleRootFileViewProvider.getPsiInner(SingleRootFileViewProvider.java:105)
	at com.intellij.psi.AbstractFileViewProvider.getPsi(AbstractFileViewProvider.java:190)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findFile(FileManagerImpl.java:363)
	at com.intellij.psi.impl.PsiManagerImpl.findFile(PsiManagerImpl.java:156)
	at lermitage.intellij.extra.icons.BaseIconProvider.getIcon(BaseIconProvider.java:174)
	at com.intellij.util.IconUtil.getProvidersIcon(IconUtil.kt:223)
	at com.intellij.util.IconUtil.computeFileIcon(IconUtil.kt:164)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.updateFileIcon(FileEditorManagerImpl.kt:587)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$MyFileTypeListener.fileTypesChanged(FileEditorManagerImpl.kt:1795)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:680)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:640)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:415)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:384)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:454)
	at jdk.proxy1/jdk.proxy1.$Proxy61.rootsChanged(Unknown Source)
	at com.intellij.openapi.roots.impl.ProjectRootManagerComponent.fireRootsChangedEvent(ProjectRootManagerComponent.kt:239)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl.fireRootsChanged(ProjectRootManagerImpl.java:483)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl$2.fireRootsChanged(ProjectRootManagerImpl.java:160)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl$2.fireRootsChanged(ProjectRootManagerImpl.java:157)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl$BatchSession.rootsChanged(ProjectRootManagerImpl.java:107)
	at com.intellij.openapi.roots.impl.ProjectRootManagerImpl$BatchSession.rootsChanged(ProjectRootManagerImpl.java:115)
	at com.intellij.openapi.roots.impl.ProjectRootManagerComponent$registerListeners$2.fileTypesChanged(ProjectRootManagerComponent.kt:153)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:680)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:640)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:415)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:394)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:454)
	at jdk.proxy1/jdk.proxy1.$Proxy154.fileTypesChanged(Unknown Source)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.fireFileTypesChanged(FileTypeManagerImpl.java:1047)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl$2.extensionAdded(FileTypeManagerImpl.java:204)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl$2.extensionAdded(FileTypeManagerImpl.java:193)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.notifyListeners(ExtensionPointImpl.java:753)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.lambda$registerExtensions$17(ExtensionPointImpl.java:957)
	at com.intellij.ide.plugins.DynamicPlugins.loadPluginWithoutProgress$lambda$29(DynamicPlugins.kt:867)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:958)
	at com.intellij.ide.plugins.DynamicPlugins.loadPluginWithoutProgress(DynamicPlugins.kt:850)
	at com.intellij.ide.plugins.DynamicPlugins.loadPlugin$lambda$26(DynamicPlugins.kt:822)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)

**Comment by jonathanlermitage** [comment]
Fix will be part of 2023.2.6 release.

**Comment by jonathanlermitage** [comment]
Here is a build with the fix: 
[Extra Icons-2023.2.5-231.zip](https://github.com/jonathanlermitage/intellij-extra-icons-plugin/files/11589189/Extra.Icons-2023.2.5-231.zip)
